lib/modules/datasource/helm/index.spec.ts

```diff
@@ -171,7 +171,7 @@ describe('modules/datasource/helm/index', () => {
       httpMock
         .scope('https://example-repository.com')
         .get('/index.yaml')
-        .reply(200, Fixtures.get('index_emptypackage.yaml'));
+        .reply(200, Fixtures.get('index_empty-package.yaml'));
```

Comment:
why rename?

Reply:
Tidyness and consistency.

I added `index_blank-digest.yaml` separating the words with a hypen for readability and to appease the spell check.

So it was nicer to make `index_emptypackage.yaml` follow suit and get rid of the spell check warning.